influunt-app/app/plugins/diagrama-intervalos.js

```diff
@@ -102,6 +104,9 @@ var influunt;
           var status = diagrama[i][0];
           var duracao = 1;
           for(var j = 0; j < diagrama[i].length; j++){
+            if(diagrama[i].length > tempoCiclo){
+              return {erros: ["Tempo de Ciclo é menor que os tempos dos estágios."]};
+            }
```

Comment:
Esse bloco de código precisa estar dentro desse `for j = 0...`? Deixando dentro esse código vai ser executado j vezes, sempre com o mesmo resultado.